.github/workflows/docker.yml

```diff
@@ -0,0 +1,35 @@
+name: Docker
+
+on: [push, pull_request]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        ps-versions: ['1.5', '1.6', '1.7', 'nightly', 'beta']
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Fetch Docker Official
+      run: git clone https://github.com/docker-library/official-images.git ~/official-images
+
+    - name: Set environment variable
+      run: echo "image=prestashop/docker-internal-images:${{ matrix.ps-versions }}" >> $GITHUB_ENV
+
+    - name: Docker Build
+      run: docker build --build-arg INSTALL_LANGUAGES=0 -t "${{ env.image }}" .
+      working-directory: ${{ matrix.ps-versions }}
+
+    - name: Docker Images
+      run: docker images
+      working-directory: ${{ matrix.ps-versions }}
```

Comment:
Is it needed or did you use it for debugging purpose?

Reply:
No it's because we need to be in the directory where the Dockerfile is.

Reply:
I mean this task, not only the `working-directory`

Reply:
Thanks. It's fixed.